### DISCUSSION TITLE
Puan kazanmak icin neler yapmalıyım?

### DISCUSSION THREAD

**Discussion by mstfcnky** [IBAX-io/go-ibax#1042] [Soru Cevap - Türkçe] [answered]
Puan kazanmak icin neler yapmalıyım?

**Comment by Yusufknk** [comment, marked as answer]
Çekilişlere katılın, retweet görevlerini yapın, sistem önerilerini githubta paylaşın.

**Reply by mmuatt86** [reply]
Sistem önerilerini gıthup tan nasıl paylaşacağız. Gıthup çoğu kişinin yabancı olduğu  bir platform

**Comment by hes91** [comment]
Tweetter sayfasında ki gönderileri dikkate al.Proje geliştirmeye destek ol ve günlük görevleri yap.Başarılar
